crates/api/Cargo.toml

```diff
@@ -40,7 +40,7 @@ tempfile = "3.1"
 maintenance = { status = "actively-developed" }
 
 [features]
-default = ['wat', 'jitdump']
+default = ['wat', 'jitdump', 'vtune']
```

Comment:
FWIW `jitdump` didn't actually require any extra C libraries or anything, so the ease of building it was why I went ahead and placed it in the default features. For the ittapi crate, however, it requires a native non-Rust toolchain, so we may want to leave this out of the default features for now while we sort out the build requirements there?